testsuite/features/profiles/mgr.suse.de/Docker/authprofile/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Container used to test Content Management feature
```

Comment:
Two suggestions about filename:

1) What would be the problem if we sticked to current hierarchy, ie

   `testsuite/features/profiles/Kiwi`
   `testsuite/features/profiles/Docker`

? No strong feelings, I'm just not seing why we need that inversion.


2) what about making that file path independant of real host's hostname?

Possible solution:

   `testsuite/features/profiles/Docker/internal_nue/authprofile/Dockerfile`

That way, if it moves e.g. to `docker.suse.de` or `docker.qa.suse.de` or whatever else, we don't need to fix the test suite.

Reply:
Well, mgr.suse.de and prv.suse.net are not the hostnames. They describe already an area where they run. Using "internal_nue", "internal_prv" and "external_aws" ... would be another way.

I think about the suggestion from @juliogonzalez to configure a common hostname in /etc/hosts and use everywhere the same hostname.

I will try this first